js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -1074,6 +1075,8 @@ var EditingController = modules.ViewController.inherit((function() {
             if(deferreds.length) {
                 that._saving = true;
 
+                dataSource && dataSource._changeLoadingCount(1);
```

Comment:
@AlekseyMartynov take a look here, please. Why do we need to use this private method?